spec/feature/guest_user_spec.rb

```diff
@@ -20,7 +23,7 @@
     end
   end
 
-  it 'can see the category items' do
+  xit 'can see the category items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.